### PR TITLE
fix: remove patently false information from readme

### PR DESCRIPTION
**Pull request by jamesonwilliams**
### [See a rendering of the updated README.md](https://github.com/aws-amplify/aws-sdk-android/blob/0cb5eb9b96f586eb4cd3ebabe215886286f4f76a/README.md#aws-sdk-for-android).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,198 +4,114 @@
...
-## Building the SDK
-This section describes how you can compile the SDK source code on your own.
-
-### Pre-requisites to build the SDK
-
-1. The AWS Core Runtime (`aws-android-sdk-core`) module requires Android API 23 only for compilation locally. Please download and install Android API Level 23 through SDK Manager in Android Studio. 
-
-2. When building the [Core module](https://github.com/aws-amplify/aws-sdk-android/blob/master/aws-android-sdk-core/build.gradle#L32), it requires the following:
-    1. Environment variable `ANDROID_HOME` to be set in order to find the `android-23.jar`.
-    1. Environment variable `ANDROID_PLATFORM` to be set to the platform number (`10` through `28` and above)
```
Comment (raphkim) [inline README.md:174]: Is this no longer relevant? I remember these steps being essential to building the SDK.

File: README.md [hunk 2]
```diff
@@ -4,198 +4,114 @@
...
 * Unless explicitly stated, calls are synchronous and must be taken off of the main thread.
 * Unless explicitly stated, calls can always throw an AmazonServiceException or an AmazonClientException (depending on if the exception is generated by the service or the client respectively).
 * The SDK will handle re-trying requests automatically, but unless explicitly stated will throw an exception if it cannot contact AWS.
 * We are always looking to help, please feel free to open an [issue](https://github.com/aws-amplify/aws-sdk-android/issues).
 
-## Building the SDK
-This section describes how you can compile the SDK source code on your own.
-
-### Pre-requisites to build the SDK
-
```
Comment (TrekSoft) [inline README.md:169]: These pre-reqs are no longer needed?

Comment (jamesonwilliams) [reply README.md:169]: Ah yea, it is. I'll add this back.

File: README.md (hunk 1 above)
Comment (raphkim) [reply README.md]: NEVERMIND DAVID IS FAST

**Pull request merged**